src/index.js

```diff
@@ -96,7 +98,7 @@ async function migrate (path, { toVersion, ignoreLock = false, repoOptions, onPr
       counter++
       log(`Migrating version ${migration.version}`)
       try {
-        if (!isDryRun) await migration.migrate(path, repoOptions, isBrowser)
+        if (!isDryRun) await migration.migrate(path, repoOptions, IS_BROWSER)
```

Comment:
True, I originally was not that familiar with the `browser` field, so I went in this way. But if it will be properly documented I think this solution will be better, especially since most of the migrations should not need to differ between browser and node environments.